CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 2.3.2 / 2018-07-12
+
+* [BUGFIX] Fix various tsdb bugs (#4369)
+* [BUGFIX] Reorder startup and shutdown to prevent panics. (#4321)
+* [BUGFIX] Exit with non-zero code on error (#4296)
+* [BUGFIX] discovery/kubernetes/ingress: fix scheme discovery (#4329)
+* [BUGFIX] Fix race in zookeeper sd (#4355)
+* [BUGFIX] Better timeout handling in promql (#4291 #4300)
+* [BUGFIX] Bubble up errors to promql from populating iterators (#4136)
```

Comment:
From a user's perspective it's not really clear what (user-visible) bug has been fixed here. Maybe something about how there was a silently ignored error resulting in partial data for a PromQL query?

Reply:
I've tweaked it. This is a very subtle thing that I'd be surprised if anyone has ever hit, I don't think it needs much detail here.